modules/mvt/src/lib/parse-mvt.js

```diff
@@ -13,33 +13,32 @@ import Protobuf from 'pbf';
 export default function parseMVT(arrayBuffer, options) {
   options = normalizeOptions(options);
 
-  if (arrayBuffer.byteLength === 0) {
-    return [];
-  }
-
-  const tile = new VectorTile(new Protobuf(arrayBuffer));
-  const loaderOptions = options.mvt;
   const features = [];
 
-  const selectedLayers = Array.isArray(loaderOptions.layers)
-    ? loaderOptions.layers
-    : Object.keys(tile.layers);
+  if (arrayBuffer.byteLength > 0) {
+    const tile = new VectorTile(new Protobuf(arrayBuffer));
+    const loaderOptions = options.mvt;
 
-  selectedLayers.forEach(layerName => {
-    const vectorTileLayer = tile.layers[layerName];
-    const featureOptions = {...loaderOptions, layerName};
+    const selectedLayers = Array.isArray(loaderOptions.layers)
+      ? loaderOptions.layers
+      : Object.keys(tile.layers);
 
-    if (!vectorTileLayer) {
-      return;
-    }
+    selectedLayers.forEach(layerName => {
+      const vectorTileLayer = tile.layers[layerName];
+      const featureOptions = {...loaderOptions, layerName};
 
-    for (let i = 0; i < vectorTileLayer.length; i++) {
-      const vectorTileFeature = vectorTileLayer.feature(i);
+      if (!vectorTileLayer) {
+        return;
+      }
 
-      const decodedFeature = getDecodedFeature(vectorTileFeature, featureOptions);
-      features.push(decodedFeature);
-    }
-  });
+      for (let i = 0; i < vectorTileLayer.length; i++) {
+        const vectorTileFeature = vectorTileLayer.feature(i);
+
+        const decodedFeature = getDecodedFeature(vectorTileFeature, featureOptions);
+        features.push(decodedFeature);
+      }
+    });
+  }
```

Comment:
It looks like you added this so that an empty features array could still be passed to `binaryToGeoJson`. I think it would be clearer if you refactored this body into a separate function that parses the MVT to GeoJSON.

I.e.
```js
const features = _parseToGeoJSON(arrayBuffer, options);

if (options.gis.format === 'binary') {
  return geojsonToBinary(features);
}

return features;
```

Then `_parseToGeoJSON` can be exactly the same as before this PR.